docs/reference/snapshot-restore/repository-azure.asciidoc

```diff
@@ -2,7 +2,7 @@
 === Azure repository
 
 You can use https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction[Azure Blob storage] as a repository for
-{ref}/modules-snapshots.html[Snapshot/Restore].
+{ref}/snapshot-restore.html[Snapshot/Restore].
```

Comment:
For some reason, GitHub isn't letting me leave a suggestion, but I'd switch this to an internal xref:

```
<<snapshot-restore,Snapshot/Restore>>.
```

Reply:
Edited to internal ref